### PR TITLE
added smoketest option to run smoke tests in retest.py

### PR DESCRIPTION
**Pull request by NguyenNhuDi**
Added a new smoketest test set in rtest.xml so that smoke tests can be executed with rtest.py --test smoketest

### CODE REVIEW

File: rtest.xml [hunk 1]
```diff
@@ -2,10 +2,14 @@
...
 <test sets="psdb">
   <run name="all_tests">{CTEST_FILTER} {CTEST_REGEX}</run>
 </test>
 <test sets="osdb">
   <run name="all_tests">{CTEST_FILTER} {CTEST_REGEX}</run>
 </test>
+<test sets="smoketest">
+  <run name="smoke_tests">{SMOKE_TEST}</run>
+</test>
```
Comment (TorreZuk) [inline rtest.xml]: Please rename test set as just "smoke" as that is the name convention set up by rocblas (and in documentation -t smoke)

Comment (NguyenNhuDi) [reply rtest.xml]: done!

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Documentation for rocPRIM is available at
 ## Unreleased rocPRIM-3.3.0 for ROCm 6.3.0
 
 ### Additions
-
+* Add --test smoke option in retest.py. It will run a subset of tests such that the total test time is in 5 minutes. Use python3 ./rtest.py --test smoke or python3 ./rtest.py -t smoke to execute smoke test.
```
Comment (TorreZuk) [inline CHANGELOG.md]: Thanks for the consistency fix.   Sorry missed what expect is a typo "retest.py" you mean "rtest.py" right?

Comment (NguyenNhuDi) [reply CHANGELOG.md]: Thanks for pointing this out! Its been updated

**Review by stanleytsang-amd — APPROVED** [review]

**Pull request merged**